lib/nylas/raw_message.rb

```diff
@@ -11,8 +11,12 @@ def initialize(mime_compatible_string, api:)
     end
 
     def send!
-      Message.new(api.execute(method: :post, path: "/send", payload: mime_compatible_string,
-                              headers: { "Content-type" => "message/rfc822" }).merge(api: api))
+      Message.new(**api.execute(
```

Comment:
Just confirming: The only change here is the double splat, correct? The reformatting makes it a little challenging to see.

Reply:
Yes.